linode/manager
# Comment out cost estimator

## Description

Comments out and disables cost estimator on ImageCreateTab.

## How to test

Navigate to image creation. Ensure that there is no price estimator on the page.
